v0.0.1/tasks-schema.json

```
@@ -0,0 +1,957 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/main/v0.0.1/tasks-schema.json",
+  "title": "Hub model task definitions",
+  "type": "object",
+  "properties": {
+    "schema_version": {
+      "type": "string",
+      "format": "uri"
+    },
+    "rounds": {
+      "description": "Array of modeling round properties",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "Individual modeling round properties",
+        "properties": {
+          "round_id_from_variable": {
+            "description": "Whether the round identifier is encoded by a task id variable in the data.",
+            "type": "boolean"
+          },
+          "round_id": {
+            "description": "Round identifier. If round_id_from_variable = true, round_id should be the name of a task id variable present in all sets of modeling task specifications",
+            "examples": ["round-1", "2022-11-05", "origin_date"],
+            "type": "string"
+          },
+          "round_name": {
+            "description": "An optional round name. This can be useful for internal referencing of rounds, for examples, when a date is used as round_id but hub maintainers and teams also refer to rounds as round-1, round-2 etc.",
+            "examples": ["round-1"],
+            "type": "string"
+          },
+          "model_tasks": {
+            "type": "array",
+            "description": "Array defining round-specific modeling tasks. Can contain one or more groups of modeling tasks per round where each group is defined by a distinct combination of values of task id variables.",
+            "items": {
+              "type": "object",
+              "properties": {
+                "task_ids": {
+                  "description": "Group of valid values of task id variables. A set of valid tasks corresponding to this group is formed by taking all combinations of these values.",
+                  "type": "object",
+                  "properties": {
+                    "origin_date": {
+                      "description": "An object containing arrays of required and optional unique origin dates. Origin date defines the starting point that can be used for calculating a target_date via the formula target_date = origin_date + horizon * time_units_per_horizon (e.g., with weekly data, target_date is calculated as origin_date + horizon * 7 days)",
```

Noting that the *'s aren't showing up in the docson, b/c they are being interpreted as markdown italics. Maybe change to x's?
```suggestion
                      "description": "An object containing arrays of required and optional unique origin dates. Origin date defines the starting point that can be used for calculating a target_date via the formula target_date = origin_date + horizon x time_units_per_horizon (e.g., with weekly data, target_date is calculated as origin_date + horizon x 7 days)",
```